app/Jobs/MutePromotions/ImportMutePromotions.php

```diff
@@ -35,22 +34,18 @@ public function __construct($record, ImportFile $importFile)
     }
 
     /**
-     * Execute the job to set user's promotions_muted_at field.
-     *
-     * @return array
+     * Execute the job to mute user promotions.
      */
     public function handle()
     {
-        $user = gateway('northstar')->asClient()->updateUser($this->userId, [
-            'promotions_muted_at' => Carbon::now(),
-        ]);
+        gateway('northstar')->asClient()->delete('v2/users/'. $this->userId . '/promotions');
```

Comment:
The return value here is a boolean, so I changed this up.

Reply:
In Gateway, we return `true` or `false` for delete requests to indicate whether they were successful or not. In hindsight, I'm not sure this was the best API decision to make across the board like this but it's what we've got! 😅

Reply:
Ah, I see. I think we may want to change this slightly to throw an error in this function if the response is `false`, so we can retry any failed jobs (or inspect why they failed)

Reply:
I see that Gateway is looking to see a `success` property in [the API response](https://github.com/DoSomething/gateway/blob/dec50f36ecdbd1023e7346c0e261dcb4e81ca5dd/src/Common/RestApiClient.php#L313-L316) of a delete request when determining the return value. It feels kinda icky but I suppose we can modify the `DELETE /users/:id/promotions` response to return both `data` and `success` properties so we'll be able to know when requests fail?

e.g.
```
{
    "success": true,
    "data": {
        "id": "600201a023a8223a1e4575a7",
        "display_name": "Virgie D.",
        "first_name": "Virgie",
        "last_initial": "D",
        "photo": null,
        ...
}
````